tools/sof-kernel-log-check.sh

```diff
@@ -3,9 +3,23 @@
 begin_line=${1:-1}
 declare err_str ignore_str project_key
 err_str="error|failed|timed out|panic|oops"
-ignore_str="error: debugfs write failed to idle -16|error: status|iteration [01]"
+ignore_str=""
 project_key="sof-audio"
 
+# There will be debug logs at each failed initializaiton of DSP:
+#   sof-audio-pci 0000:00:1f.3: error: status = 0x00000000 panic = 0x00000000
+#   sof-audio-pci 0000:00:1f.3: error: Error code=0xffffffff: FW status=0xffffffff
+#   sof-audio-pci 0000:00:1f.3: error: iteration 0 of Core En/ROM load failed: -110
+# We will reinit DSP if it is failed to init, and retry times is 3, so the errors in
+# debug logs at the frist and second retry can be ignored.
+# Check https://github.com/thesofproject/linux/pull/1676 for more information.
+# TODO explain for "error: Error ..."
+ignore_str="$ignore_str"|'error: iteration [01]'
+ignore_str="$ignore_str"|'error: status'
```

Comment:
I think you can add for ignore `error: status`, which is not really error, the error will report by other content

Reply:
@Bin-QA can you point at the code that prints `error: status`? Thanks!

Reply:
As @marc-hb pointed here: https://github.com/thesofproject/sof-test/issues/261#issuecomment-649837495:
- `error: status` added by https://github.com/thesofproject/sof-test/commit/50f6ccf895c5a70ba071cd21b4fe15ce3090d74a, mentioned https://github.com/thesofproject/linux/pull/1676
- `error: iteration [01]` added by https://github.com/thesofproject/sof-test/commit/50f6ccf895c5a70ba071cd21b4fe15ce3090d74a mentioned https://github.com/thesofproject/sof-test/issues/43

Take a look at https://github.com/thesofproject/sof-test/issues/43, you should see it also points to https://github.com/thesofproject/linux/pull/1676